src/kokkos/test/kokkos/OneHistoContainer_t.cc

```diff
@@ -46,7 +46,7 @@ void go() {
 
     using TeamHist = cms::kokkos::HistoContainer<T, NBINS, N, S, uint16_t>;
 
-    Kokkos::View<TeamHist, KokkosExecSpace> histo_d("histo_d");
+    Kokkos::View<TeamHist[1], KokkosExecSpace> histo_d("histo_d");
```

Comment:
Why was this change needed?

Reply:
In `Hist::finalize()`, each hist view has to take an argument, e.g. `update += histo(leagueRank).off[i];`. This expression cannot be applied to "scalar" view. For example,
```c++
Kokkos::View<TeamHist, KokkosExecSpace> histo_d("histo_d");
Kokkos:: parallel_scan( ...
  update += histo(0).off[i]; // this will not work, result in compile time error
  update += histo().off[i]; // this will work, "scalar" view takes no argument
);
```
Since the original `forEachInWindow` function works over a scalar view only and the view in `finalize()` has to take an argument, I just make a minimal change to make both of them run. Let me know if a thorough solution to this is required.

Reply:
Thanks. Just to check that I understood, the need to be able to call `HistoContainer::finalize()` for an array of `HistoContainer` necessitates the indexing via `histo(leagueRank)`, and therefore all calls need to look like arrays (i.e. mimic a scalar with 1-element array). 

Then I'm wondering why
https://github.com/cms-patatrack/pixeltrack-standalone/blob/95d56aad8feb461165fe7563bedc564ac3c4f816/src/kokkos/KokkosCore/HistoContainer.h#L349
takes a scalar and not `Kokkos::View<Histo*, ExecSpace>`?

Reply:
Oh no, that's a mistake. Thanks for pointing this out.